suricata.yaml.in

```diff
@@ -660,6 +660,56 @@ af-packet:
     #use-mmap: no
     #tpacket-v3: yes
 
+dpdk:
+  eal-params:
+    proc-type: primary
+    a: 0000:3b:00.0
+    l: 0-1
+
+  # DPDK capture support
+  # Currently queues should be assigned to cores in ratio in 1:1 ratio
+  interfaces:
+    - interface: 0000:3b:00.0 # PCIe address of the NIC port
+      threads: auto # or number of threads - auto takes all lcores, no thread entry takes all cores
+      rx-queues: auto # or number of queues - auto assigns each core 1 queue
+      tx-queues: auto # or number of queues - auto assigns each core 1 queue
+      promisc: true
+      multicast: true # enables also detection on multicast packets
+      checksum-checks: true # if Suricata should validate checksums
+      checksum-checks-offload: true # if possible offload checksum validation to the NIC
+      mtu: 3000 # Set MTU of the device in bytes # default 1526
+      mempool-size: 65356 # 64*1024
+      mempool-cache-size: 250
+      rx-descriptors: 1024
+      tx-descriptors: 1024
+      # IPS mode for Suricata works in 3 modes - none, tap, ips
+      # - none: disables IPS functionality (does not further forward packets)
+      # - tap: forwards all packets and generates alerts (omits DROP action)
```

Comment:
as mentioned above DPDK TAP has a different meanings, please use alternative name to represent this behaviour (rework)

Reply:
Please cehck your output of pkg-config and rework, you are missing the point.

Reply:
I have added additional explanation to the tap mode:
This is not DPDK TAP

I think this states clear that the setting of the copy mode is bound to Suricata and thus setting TAP mode to copy mode does not enable any functionality of DPDK TAP interface.